packages/cli/src/lib/helpers/schema.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
looks great! Let's move this into its own package so that all other packages can use it (i.e. all clients & the CLI). Maybe call the package `manifest-schema`